cudo.js

```diff
@@ -29,21 +29,25 @@ async function main() {
                     description: 'project directory'
                 })
         }, initCmd)
-        .command('compile [contractname]', 'compile smart contract', (yargs) => {
-            yargs.positional('contractname', {
+        .command('compile [contractsFolderName]', 'Compiles in alphabetical order the smart contracts in the specified contracts folder(by default: workspace/contracts). \n Please note that your contracts have to be in the [workspace] folder. \n If you decide to change the default name - you need to change the workspace Cargo.toml file to point to the your folder name. \n Example usage: cudo compile myContractsFolderName.', (yargs) => {
```

Comment:
Description seems a bit too long. We can add details about the workspace in the README.md of the project.